include/xtensor/xadapt.hpp

```diff
@@ -404,6 +404,28 @@ namespace xt
         return return_type(std::move(buf), shape, l);
     }
 
+    /**
+     * @copydoc adapt(P&&, typename A::size_type, O, const SC&, layout_type, const A&)
+     */
+    template <layout_type L = XTENSOR_DEFAULT_LAYOUT, class P, class O, std::size_t N, class A = detail::default_allocator_for_ptr_t<P>>
+    inline xtensor_adaptor<xbuffer_adaptor<xtl::closure_type_t<P>, O, A>, N, L> adapt(
+        P&& pointer,
+        typename A::size_type size,
+        O ownership,
+        const typename A::size_type (&shape)[N],
+        layout_type l = L,
+        const A& alloc = A()
+    )
+    {
+        (void) ownership;
```

Comment:
Just a question: why is this parameter not used? How could one use `xt::acquire_ownership`